circus-train-api/src/main/java/com/hotels/bdp/circustrain/api/copier/CopierContext.java

```diff
@@ -50,6 +56,40 @@ public CopierContext(
     }
     this.replicaLocation = replicaLocation;
     this.copierOptions = ImmutableMap.copyOf(copierOptions);
+    this.sourceTable = sourceTable;
+    this.sourcePartitions = sourcePartitions;
+  }
+
+  public CopierContext(
```

Comment:
Can we just remove these three constructors and stick with changing this one. It's very confusing what you need to actually make. We're not using